util/src/main/java/io/kubernetes/client/informer/ResyncRunnable.java

```diff
@@ -0,0 +1,32 @@
+package io.kubernetes.client.informer;
+
+import io.kubernetes.client.informer.cache.Store;
+import io.kubernetes.client.informer.cache.function.ResyncFunc;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ResyncRunnable<ApiType> implements Runnable {
```

Comment:
I'd love to see a JavaDoc for each of these classes explaining how they fit together for the benefit of future developers.

Reply:
yeah and i updated other classes w/ more java docs. ptal